src/components/codeBubble/codeBubble.tsx

```diff
@@ -0,0 +1,68 @@
+import './styles.css'
+import { useState } from 'react'
+import { Prism } from 'react-syntax-highlighter';
+import { tomorrow } from 'react-syntax-highlighter/dist/esm/styles/prism';
+
+interface Props {
+    codeChunk: string
+}
+
+const CodeBubble: React.FC<Props> = ({ codeChunk }) => {
+    const [copied, setCopied] = useState(false);
+
+    // remove the first and last lines
+    const code = codeChunk.trim().split('\n').slice(1, -1).join('\n')
+
+    // extract the coding language
+    const lang = codeChunk.match(/```.+\n/);
+    const language = lang ? lang[0].substring(3).trim() : '';
+
+    const handleCopy = async () => {
+        try {
+            await navigator.clipboard.writeText(code);
```

Comment:
Does this work right out of the gate or do we need to enable some sort of permission?